libraries/AP_Scripting/examples/fw_vtol_failsafe.lua

```diff
@@ -8,6 +8,10 @@
 -- If the aircraft drops below a predetermined minimum altitude, QLAND mode is engaged and the aircraft lands at its current position.
 -- If the aircraft arrives within Q_FW_LND_APR_RAD of the return point before dropping below the minimum altitude, it should loiter down to the minimum altitude before switching to QRTL and landing.
 
+-- constants
+local RC_OPTION = {TECS_PROP_FAILED=177}
```

Comment:
Isn't this 178?

Reply:
Oof, great catch. This was missed in the rebase.